<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialBlock</class>
 <widget class="QWidget" name="DialBlock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>73</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
       <property name="minimum">
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>LOW</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="minimum">
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>HIGH</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
       <property name="minimum">
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDial" name="dial"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>LIN</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>LOG</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>doubleSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DialBlock</receiver>
   <slot>lowChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_3</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DialBlock</receiver>
   <slot>valueChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBox_2</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DialBlock</receiver>
   <slot>highChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>DialBlock</receiver>
   <slot>dialTurned(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>DialBlock</receiver>
   <slot>sliderMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>DialBlock</receiver>
   <slot>linSelected(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>DialBlock</receiver>
   <slot>logSelected(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DialBlock</receiver>
   <slot>xPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialBlock</receiver>
   <slot>dialTurned(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialBlock</receiver>
   <slot>sliderMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>DialBlock</receiver>
   <slot>nameChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>lowChanged(double)</slot>
  <slot>highChanged(double)</slot>
  <slot>valueChanged(double)</slot>
  <slot>linSelected(bool)</slot>
  <slot>logSelected(bool)</slot>
  <slot>dialTurned(int)</slot>
  <slot>sliderMoved(int)</slot>
  <slot>xPressed()</slot>
  <slot>nameChanged()</slot>
 </slots>
</ui>
